rules/network.js

```diff
@@ -75,19 +75,25 @@ function _system_update_ip(name, iface) {
 };
 
 function _current_active_connection() {
-  runShellCommand('ip route get {} | grep -oP \'dev\\s+\\K[^ ]+\''.format(checkAddress), {
+  runShellCommand('ip route get {} 2>/dev/null | grep -oP \'dev\\s+\\K[^ ]+\''.format(checkAddress), {
     captureOutput: true,
     exitCallback: function (exitCode, capturedOutput) {
-      if (exitCode === 0) {
-        dev.network["Default Interface"] = capturedOutput;
-      }
+      dev.network["Default Interface"] = exitCode === 0 ? capturedOutput : "";
     }
   });
-  runShellCommand('nmcli -t -f NAME c s -a', {
+
+  runShellCommand("nmcli -t -f DEVICE,NAME c s -a 2>/dev/null", {
     captureOutput: true,
     exitCallback: function (exitCode, capturedOutput) {
-      if (exitCode === 0) {
-        dev.network["Active Connection Name"] = capturedOutput;
+      if (exitCode != 0) {
+        dev.network["Active Connection Name"] = "";
+        return;
+      }
+      var lines = capturedOutput.split("\n");
+      for (var i = 0; i < lines.length; i++) {
+        if (lines[i].indexOf(dev.network["Default Interface"] + ":") === 0) {
+          dev.network["Active Connection Name"] = lines[i].split(":")[1];
```

Comment:
```suggestion
          dev.network["Active Connection Name"] = lines[i].split(":")[1].trim();
```

Ну чтобы уж наверняка

Reply:
поправил